src/App.tsx

```diff
@@ -2,166 +1,105 @@
+// import { Loader } from './components/Loader';
 
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">There are no people on the server</p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import { Routes, Route, Link, Navigate } from 'react-router-dom';
+import cN from 'classnames';
+import { useLocation } from 'react-router-dom';
+import { PeopleTable } from './components/PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from './types';
+import { getPeople } from './api';
+
+function addParents(people: Person[]) {
+  return people.map(person => {
+    return {
+      ...person,
+      mother: people.find(peopleItem => peopleItem.name === person.motherName),
+      father: people.find(peopleItem => peopleItem.name === person.fatherName),
+    };
+  });
+}
+
+export const App = () => {
+  const location = useLocation().pathname;
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(data => {
+        setPeople(addParents(data));
+      })
+      .catch(() => setHasError(true))
```

Comment:
It might be helpful to log the error in the catch block to understand what went wrong during the API call.